.github/workflows/nxlatest-e2e.yml

```diff
@@ -0,0 +1,38 @@
+name: NX_Latest_E2E
+on:
+  schedule: 
+    - cron: 0 6 * * 1
+  workflow_dispatch:
+jobs:
+  e2e:
+    permissions:
+      contents: read
+    name: E2E
+    uses: nrwl/ci/.github/workflows/nx-cloud-main.yml@v0.11.0
+    with:
+      runs-on: ubuntu-latest
+      node-version: 18
+      number-of-agents: 3
+      parallel-commands-on-agents: |
+        npx nx affected --skip-nx-cache --base=last-release --target=e2e --parallel=1
```

Comment:
Don't think nx affected is the correct command to run here. If the NX version is bumped, but we have no changes since our base (last-release), no tests will run. This means the command will pass, indicating things work on the latest version of NX, when in fact, we've not ran anything. nx run-many is what I'd recommend using here

Reply:
I was using --skip-nx-cache which did seem to be always running the tests even when nothing had changed. However I've changed this to run-many its probably more appropriate.

Reply:
In this case the `last-release` was performed two months ago, so yes, `nx affected` will work at present. But once we perform a formal release of the plugins, we would encounter the scenario I mentioned. So good that it's been swapped :D